cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/connector/resource/AzureDatabaseResourceService.java

```diff
@@ -198,13 +211,38 @@ private List<CloudResourceStatus> deleteResources(List<CloudResource> resources,
         azureGenericResources.forEach(cr -> persistenceNotifier.notifyDeletion(cr, cloudContext));
 
         return findResources(resources, List.of(AZURE_DATABASE)).stream()
-                .map(r -> deleteDatabaseServerAndNotify(r, cloudContext, client, persistenceNotifier, force))
+                .map(r -> deleteDatabaseServerAndNotify(r, cloudContext, client, persistenceNotifier, force, keyVaultUrl, keyVaultResourceGroupName))
                 .collect(Collectors.toList());
     }
 
     private CloudResourceStatus deleteDatabaseServerAndNotify(
-            CloudResource cloudResource, CloudContext cloudContext, AzureClient client, PersistenceNotifier persistenceNotifier, boolean force) {
+            CloudResource cloudResource, CloudContext cloudContext, AzureClient client, PersistenceNotifier persistenceNotifier, boolean force,
+            String keyVaultUrl, String keyVaultResourceGroupName) {
         LOGGER.debug("Deleting postgres server {}", cloudResource.getReference());
+        if (keyVaultUrl != null) {
```

Comment:
Perhaps add an INFO log in the associated `else` branch of `if (keyVaultUrl != null)` for better tracking, printing both the `keyVaultUrl` and `keyVaultResourceGroupName`.